js/utils/helpers.js

```diff
@@ -1938,6 +1938,19 @@
                                         }
 
                                         // handles `style="some: style;"` case from template
+                                        // min/max width needs to be applied to the wrapper of the caption and fullscreen button for consistent button placement
+                                        // check for min-width style
+                                        var minWidthIdx = attr[1].indexOf("min-width");
+                                        if (minWidthIdx >= 0) {
+                                            widthStyles += attr[1].substring(minWidthIdx, attr[1].indexOf(";", minWidthIdx)+1);
```

Comment:
If `min-width` is the last style defined and therefore `;` is missing, this will not work properly (`attr[1].indexOf(";", minWidthIdx)` returns -1 in this case). You would have to check that. 

Also I think it might be better to avoid one liners for stuff like this. It's hard to follow what you're doing.